Tests/Foundation/Tests/TestDecimal.swift

```diff
@@ -7,9 +7,6 @@
 // See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 //
 
-import Foundation
-import XCTest
-
```

Comment:
We probably need to add these back to get the macOS build working again, but overall looks good!

Reply:
Well that's just embarrassing.

Reply:
No worries at all, tests look good!